README.md

```diff
@@ -127,6 +127,7 @@ This page introduces all links related to core and ecosystem development of Klay
 * [servicechain-value-transfer-examples](https://github.com/klaytn/servicechain-value-transfer-examples)
 * [CN Staking contract tests](https://github.com/klaytn/cn-staking-contract-tests)
 * [NFT Transfer on Cypress using Metamask](https://github.com/Yeonju-Kim/nft-transfer-klaytn)
+* Soulbound token: [Contract](https://github.com/bisonai/sbt-contracts) & [Typescript interface](https://github.com/bisonai/sbt-js)
```

Comment:
```suggestion
* Soulbound token: [Contract](https://github.com/bisonai/sbt-contracts) & [Typescript interface](https://github.com/bisonai/sbt-js) [sbt-js npm](https://www.npmjs.com/package/@bisonai/sbt-js)
```

it could be good to add npm package link, too.

Reply:
Thank you for suggestion! I made a commit including links to npm packages.